utils/data.go

```diff
@@ -16,3 +19,18 @@ func UnwrapJSON(data map[string]interface{}, tag string) (map[string]interface{}
 	}
 	return data, nil
 }
+
+func GetTypedFieldFromJSON[T any](data map[string]interface{}, tag string) (T, error) {
```

Comment:
Is this used anywhere?

Reply:
I was using it in previous commits but not now, I'll remove it then